subprojects/core/src/main/java/org/gradle/api/internal/DefaultNamedDomainObjectCollection.java

```diff
@@ -463,10 +461,20 @@ private boolean isConfigureMethod(String name, Object... arguments) {
         NavigableMap<String, T> asMap();
 
         <S extends T> Index<S> filter(CollectionFilter<S> filter);
+
+        @Nullable
+        ProviderInternal<? extends T> getPending(String name);
+
+        void putPending(String name, ProviderInternal<? extends T> provider);
+
+        @Nullable
+        void removePending(String name);
```

Comment:
`@Nullable` on a method usually means that the return type is nullable. But the return type here is `void` is this what you meant?

Reply:
Heh, these originally returned the provider that was removed, but then I changed them and forgot to remove the annotation.  Removed.